bio_files_processor.py

```diff
@@ -0,0 +1,89 @@
+import os
+from dataclasses import dataclass
+
+def convert_multiple_fasta_to_oneline(input_fasta: str, output_fasta: str = 'convert_multiple_fasta_to_oneline-output'):
+    """
+    Read .fasta file where swquences are written on multiple lines,
+    and create a .fasta file where each sequence is contained in one string
+    arguments:
+    - input_fasta (str): path to input file
+    - output_fasta (str): name of output .fasta file
+    return:
+    - no return
+    """
+    with open(input_fasta) as fasta:
+        fasta_dict = {}
+        current_name = fasta.readline().strip()
+        fasta_dict[current_name] = []
+        for line in fasta:
+            if line.startswith('>'):
+                fasta_dict[current_name] = ''.join(fasta_dict[current_name])
+                current_name = line.strip()
+                fasta_dict[current_name] = []
+            else:
+                fasta_dict[current_name].append(line.strip())
+        fasta_dict[current_name] = ''.join(fasta_dict[current_name])
```

Comment:
```suggestion
        fasta_dict[current_name] = ''.join(fasta_dict[current_name])
        
```
такие штуки лучше пустой строкой разделять